pandas/tests/sparse/test_groupby.py

```diff
@@ -22,29 +22,29 @@ def setup_method(self, method):
     def test_first_last_nth(self):
         # tests for first / last / nth
         sparse_grouped = self.sparse.groupby('A')
-        dense_grouped = self.dense.groupby('A')
+        dense_grouped = self.dense.to_sparse().groupby('A')
```

Comment:
This is basically just testing `a == a` :) The `to_sparse` needs to be *after* the operation.